org.gnome.OTPClient.json

```diff
@@ -0,0 +1,83 @@
+{
+  "app-id": "org.gnome.OTPClient",
```

Comment:
Is it going to be official GNOME app?

Reply:
err, no! I didn't know that the "org.gnome" is reserved for official gnome apps. I'll change it.

Reply:
Technically it's not, but I guess it makes more sense.

Reply:
Yeah - unless you're part of gnome (hosted there, a foundation member, part of an official gnome release) I really wouldn't use that namespace. Looks like you already have a handy com.github.paolostivanin.OTPClient namespace as well ;-0

Reply:
good suggestion :)